assets/src/blocks/Submenu/BackTop.js

```diff
@@ -0,0 +1,18 @@
+import useScrollPosition from '@react-hook/window-scroll';
+
+export const BackTop = ({}) => {
+  const cookies = document.getElementById('set-cookie');
+
+  const cookiesVisible = cookies && cookies.style.display !== 'none';
+
+  const scrollPosition = useScrollPosition(60);
```

Comment:
Isn't it possible to use `window.scrollY` here?

Reply:
Then how will it update if the scroll position changes?